aws/lambda/blueprint.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: AWS Lambda and Credentials
+  description: |
+    This blueprint will configure one AWS Lamba application and one AWS credential set.
+  author: XebiaScrubs
+  version: 1.0
+parameters:
+- name: AwsInfrastructureDirectory
+  type: Input
+  description: "Infrastructure directory (default: AmazonAWS)"
```

Comment:
Please stick to one template style, strings doesn't need `"` or `'` quote around in Yaml. I can see a mix of all here